time_tracker_api/time_entries/time_entries_repository.py

```diff
@@ -196,6 +199,40 @@ def find_all(
             abort(HTTPStatus.NOT_FOUND, "Time entry not found")
         return time_entries
 
+    def find_all_v2(
+        self,
+        event_context: EventContext,
+        owner_id_list: List[str],
```

Comment:
We also need the pass a list of ids of time-entries.

Reply:
I can add it